File: CodeForces/1601-1700/1669/A/1669A.rb
# Problem 1669A: Division?
# https://codeforces.com/contest/1669/problem/A
# Executed with: ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [x86_64-linux-gnu]
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: October 23rd, 2025
# Codeforces language used: Ruby 3.2.2
# 
w = gets.to_i

for counter in 1..w
    input = gets.to_i
    if input >= 1900
        puts "Division 1"
    elsif input >= 1600
        puts "Division 2"
    elsif input >= 1400
        puts "Division 3"
    else
        puts "Division 4"
    end
end

File: CodeForces/1601-1700/1669/A/1669A.pl
#!/usr/bin/perl

#! Problem 1669A: Division?
#! https://codeforces.com/contest/1669/problem/A
#! Executed with: perl 5, version 40, subversion 1 (v5.40.1) built for aarch64-linux-gnu-thread-multi
#! Linux version: Linux kernel 6.17.0-6-generic
#! Submitted on: October 27th, 2025
#! Codeforces language used: Perl 5.20.1

use strict;
use warnings;

chomp(my $runs = <STDIN>);
for (my $run = 0; $run < $runs; $run++) {
    chomp(my $n = <STDIN>);

    if ($n >= 1900) {
        print "Division 1\n";
    } elsif ($n >= 1600) {
        print "Division 2\n";
    } elsif ($n >= 1400) {
        print "Division 3\n";
    } else {
        print ("Division 4\n");
    }
}
